examples/bert/run_pretraining.py

```diff
@@ -32,7 +33,21 @@
 )
 
 flags.DEFINE_string(
-    "saved_model_output", None, "Output directory to save the model to."
+    "saved_model_output",
+    None,
+    "Output directory to save the model to.",
+)
+
+flags.DEFINE_string(
+    "checkpoint_save_path",
```

Comment:
checkpoint_directory? to give hint this should be a directory not a filepath?

Reply:
Yea, I updated the code to `checkpoint_save_dir`

Reply:
Use the whole word `directory` to agree with arg naming in `create_sentence_split_data`

Reply:
done